style.css

```
@@ -0,0 +1,24 @@
+body {
+  background-image: url("./images/bg.jpeg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  font-family: "Varela Round", sans-serif;
+}
+
+h1 {
+  font-family: "Pacifico", cursive;
+  text-align: center;
+  font-size: 72px;
+}
+
+input{
+  display: block; 
+  margin-bottom: 20px; 
+  margin:0 auto 20px; 
+  padding:10px;
+}
+
+.form{
+  width: 50%; 
+  margin: 0 auto;
+}
```

css 파일도 new line 추가해 주세요.
```suggestion
}

```